packages/echarts/tests/echarts.e2e.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { expect } from '@playwright/test';
+import { regressionTest } from './utils/test';
+
+regressionTest.describe('echarts', () => {
+  regressionTest('basic', async ({ page }) => {
+    await page.goto('basic.html');
+
+    expect(await page.screenshot({ fullPage: true })).toMatchSnapshot();
```

Comment:
Use not deprecated style -> https://playwright.dev/docs/test-snapshots#introduction
```ts
await expect(page).toHaveScreenshot();
```